sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -20,15 +17,23 @@ Install the Azure Form Recognizer client library for .NET with [NuGet][nuget]:
 dotnet add package Azure.AI.FormRecognizer
 ``` 
 
-> Note: This version of the client library defaults to the `v2.1` version of the service.
+> Note: This version of the client library defaults to the `2021-09-30-preview` version of the service.
 
 This table shows the relationship between SDK versions and supported API versions of the service:
 
 |SDK version|Supported API version of service
 |-|-
-|3.0.0 | 2.0
-|3.0.1 | 2.0
-|3.1.X | 2.0, 2.1
+|4.0.0-beta.1 | 2.0, 2.1, 2021-09-30-preview
+|3.1.X        | 2.0, 2.1
+|3.0.X        | 2.0
+
+> Note: Starting with version `2021-09-30-preview`, a new set of clients were introduced to leverage the newest features of the Form Recognizer service. Please see the Migration Guide for detailed instructions on how to update application code from client library version `3.1.X` or lower to the latest version. Additionally, see the [Changelog][formreco_changelog] for more detailed information. The below table describes the relationship of each client and its supported API version(s):
```

Comment:
Just FYI I think for Python we're planning on adding a link to the migration guide as soon as it's merged.

Reply:
I intend to do the same once we have the migration guide in .NET.